src/Controller/PresentationController.php

```diff
@@ -37,36 +28,39 @@ public function __construct(PresentationService $presentationService, Translator
     /**
      * @ApiDoc(
      *  resource=true,
-     *  description="Tido text API item page resource",
+     *  description="Tido text API annotationCollection resource",
      *  requirements={
      *      {"name"="id", "dataType"="string", "required"=true, "description"="work identifier"}
      *  }
      * )
      */
-    public function itemAction(string $id): View
+    public function annotationCollectionAction(string $id): View
```

Comment:
Do we even talk about actions in Symfony 5?

Reply:
Good take. Fixed